eks-public-cluster.tf

```diff
@@ -0,0 +1,135 @@
+# Define a KMS main key to encrypt the EKS cluster
+resource "aws_kms_key" "eks-public" {
+  description         = "EKS Secret Encryption Key"
+  enable_key_rotation = true
+}
+
+# EKS Cluster definition
+module "eks-public" {
+  source       = "terraform-aws-modules/eks/aws"
+  version      = "18.29.0"
```

Comment:
Don't forget to track this with updatecli (like it's done for the other cluster). WDYT about an upcoming PR (to avoid failing checks)?